common/djangoapps/util/organizations_helpers.py

```diff
@@ -88,6 +88,16 @@ def get_course_organizations(course_id):
     return organizations_api.get_course_organizations(course_id)
 
 
+def get_course_organization_id(course_id):
+    """
+    Returns organization id for course or None if the course is not linked to an org
+    """
+    course_organization = get_course_organizations(course_id)
```

Comment:
Is this making an API request of a Database call? 

Isn't the organization ID part of the course key?

Reply:
When I grab the org from the course_key i just get the org name (i.e. testorg ) but the get_course_organization_id is returning an org_id (i.e. 1 ). is it necessary to change the preexisting logic to try to lookup the org_id through the course_key somehow?

Reply:
No need to change existing logic. Just wanted to be sure we weren't making an API/DB call unnecessarily.